qemu/tests/usb_storage.py

```diff
@@ -124,14 +123,14 @@ def _get_output():
     @error_context.context_aware
     def _check_serial_option(serial, regex_str, expect_str):
         error_context.context("Set serial option to '%s'" % serial,
-                              logging.info)
+                              test.log.info)
```

Comment:
L101 miss `test.log.info` for `error_context.context`.  Would you mind to update it together ?

Reply:
The same as the above.